lib/senders/partials/new_device_login.html

```diff
@@ -1,6 +1,6 @@
 <% extends "partials/base/base_text_only.html" %>
 
-<% block heading %>{{t "New sign-in to Firefox"}}<% endblock %>
+<% block heading %>{{serviceHeader}}<% endblock %>
```

Comment:
Out of curiosity, why do we need to pass in the entire string as `serviceHeader` rather than passing in just a `serviceName` parameter?  The `location.html` template seems to take the second approach, e.g. using:

```
   {{#if ip }}{{t "IP address: %(ip)s" }}<br/>{{/if}}
```

But I could easily be missing something.

Reply:
I will take a look, one email changes the subject, the other one doesn't. So it was all moved to js for consistency